specification/redisenterprise/resource-manager/readme.md

```diff
@@ -85,7 +85,7 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-net-track2
```

Comment:
I have no idea if this is legit or intended. Assuming it is, great. If not... is it a bad merge?